.github/workflows/test.yml

```diff
@@ -34,10 +34,13 @@ jobs:
           sudo apt-get update
           sudo apt-get install -y gcc-aarch64-linux-gnu g++-aarch64-linux-gnu qemu-user
 
+      - name: Set up the prerequisites
+        if: runner.os == 'Windows'
+        uses: msys2/setup-msys2@v2
```

Comment:
Is this needed to run `go vet`?

Reply:
No it's need to build the abitest library since it uses GCC to compile the C code.

Reply:
Ack